src/gate_detect/src/Gate.cpp

```diff
@@ -0,0 +1,189 @@
+#include "Gate.h"
+
+Gate::Gate() {
+    lowVertThresh = 50; // Sets the maximum deviation from start to finish that
+                        // will constitute a vertical line
+    lowHorThresh = 100; // Sets the maximum deviation from start to finish that
+                        // will constitute a horizontal line
+    cannyLow = 50; // Sets the low threshold for the canny filter, adjust if dst
+                   // is not displaying all lines
+    cannyHigh = 200; // Sets the high threshold for the canny filter, adjust if
+                     // dst not displaying all lines
+    poleMax =
+    200; // Sets the minimum threshold for two lines to be called a pole
```

Comment:
I think the culprit for the odd formatting here is line 23 of the `.clang-format` file: `ContinuationIndentWidth: '0'`. Feel free to delete it, I'm not a huge fan, removing it will probably make things look nicer.

Reply:
I put all descriptions in the header files now so this shouldn't be an issue